alf/examples/sac_carla_conf.py

```diff
@@ -187,14 +176,33 @@
     optimizer=alf.optimizers.Adam(lr=learning_rate),
 )
 
-if tasac:
+if not tasac:
+    from alf.algorithms.sac_algorithm import SacAlgorithm
+    alf.config('Agent', rl_algorithm_cls=SacAlgorithm)
+
+    alf.config(
+        'SacAlgorithm',
+        actor_network_cls=actor_network_cls,
+        critic_network_cls=critic_network_cls,
+        target_entropy=partial(calc_default_target_entropy, min_prob=0.1),
+        target_update_tau=0.005,
+        critic_loss_ctor=TDLoss,
+        use_parallel_network=True,
+        use_entropy_reward=False,
+        reward_weights=reward_weights)
+else:
     from alf.algorithms.tasac_algorithm import TasacValueAlgorithm, SkipRepeatTDLoss
     value_net_cls = partial(
         alf.networks.ValueNetwork,
         fc_layer_params=(256, ),
         output_tensor_spec=env.reward_spec())
 
     alf.config('Agent', rl_algorithm_cls=TasacValueAlgorithm)
+    alf.config(
+        'TrainerConfig',
+        use_rollout_state=True,
+        temporally_independent_train_step=True)
```

Comment:
Is this the reason why the current code is much slower than before?

Reply:
yes